### PR TITLE
Edgar

### PR DESCRIPTION
**Pull request by jg-main**
Examples for the Edgar filing database

### CODE REVIEW

File: edgar-filings-examples/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+### Shares Outstanding
+
+This notebook shows how to get the shares outstanding of a company as reported in the 10-K and 10-Q filings.
+
+## Prerequisites
+
+### Database credentials
+
+For the examples to work, you will need to have credentials to access our database service, if you don't have them please feel free to contact your account executive and we'l provide it to you.
```
Comment (VasylBaliha) [inline edgar-filings-examples/README.md:34]: Maybe better to use an account manager.
we'l to we'll

File: edgar-filings-examples/notebooks/financials.ipynb
Comment (VasylBaliha) [inline edgar-filings-examples/notebooks/financials.ipynb:1]: Limit response for the last query

Comment (jg-main) [reply edgar-filings-examples/notebooks/financials.ipynb:1]: Solved

File: edgar-filings-examples/README.md (hunk 1 above)
Comment (jg-main) [reply edgar-filings-examples/README.md:34]: Will update if the access method to Arda changes.

**Pull request merged**